### PR TITLE
feat: add SideBlock and SingleToggle components

### PR DESCRIPTION
**Pull request by sofa-tata**

### CODE REVIEW

**Comment by udia76** [comment]
Generally it looks very good! well structured.
I think that we better add stories for each new component that we are adding, it is 5 minutes work now, but will save us some time in the future :)

**Comment by sofa-tata** [comment]
@udia76 sure, will do, thanks

**Comment by sofa-tata** [comment]
@udia76 done

File: lib/components/SideBlock/SideBlock.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    />
+  )
+}
+
+export const SideBlock = Template.bind({})
+SideBlock.args = {
+  name: 'Test Side Block',
+  actions: [
+    {
+      onClick: () => console.log('Delete clicked'),
```
Comment (udia76) [inline lib/components/SideBlock/SideBlock.stories.tsx:59]: If you want to simulate click you can use `import { action } from '@storybook/addon-actions'` 
This is nice to have, you can see how I did it in `Table.stories.tsx`

Comment (sofa-tata) [reply lib/components/SideBlock/SideBlock.stories.tsx]: added, thanks!

**Review by udia76 — APPROVED** [review]

**Review by mvoloshin0458 — APPROVED** [review]

**Pull request merged**